modules/publication/ajax/FileDelete.php

```diff
@@ -64,6 +65,6 @@ function userCanDelete($uploadData, $db, $user) : void
 
     if ($user->getId() !== $origUser && !in_array($user->getId(), $editors)) {
         header("HTTP/1.1 403 Forbidden");
-        return;
+        exit;
```

Comment:
I don't think existing the execution is a good idea. IMHO an error message or exception would be more appropriate.

Reply:
addressed